core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -344,6 +344,16 @@ offerbook.warning.noTradingAccountForCurrency.headline=No trading account for se
 offerbook.warning.noTradingAccountForCurrency.msg=You don't have a trading account for the selected currency.\nDo you want to create an offer with one of your existing trading accounts?
 offerbook.warning.noMatchingAccount.headline=No matching trading account.
 offerbook.warning.noMatchingAccount.msg=You don't have a trading account with the payment method required for that offer.\nYou need to setup a trading account with that payment method if you want to take this offer.\nDo you want to do this now?
+offerbook.warning.makerHasNoMatureAccountForBuyOffer=You cannot create an offer because you do not have a payment account which was created before March 15th 2019. \
```

Comment:
Not show-stopping but I made some minor changes:

```
offerbook.warning.noMatchingAccount.msg=To take this offer, you will need to set up a payment account using this payment method.\n\nWould you like to do this now?
offerbook.warning.makerHasNoMatureAccountForBuyOffer=You cannot create this offer because you do not have a payment account which was created before March 15th 2019. \
 The selected payment method is considered risky for bank chargebacks. We needed to deploy this restriction as a short-term measure for enhanced security.\n\n\
 The next software release will provide more robust chargeback protection tools, and this restriction for new accounts will be removed.
offerbook.warning.riskyBuyOfferWithImmatureAccountAge=This offer cannot be taken because the maker's payment account \
 was created after March 15th 2019, and the payment method is considered risky for bank chargebacks. We needed to deploy this restriction as a \
 short-term measure for enhanced security.\n\n\
 The next software release will provide more robust protection tools so that offers with this risk profile can be traded again.
offerbook.warning.sellOfferAndAnyTakerPaymentAccountForOfferMature=This offer cannot be taken because your payment account \
 was created after March 15th 2019 and the payment method is considered risky for bank chargebacks. We needed to deploy this restriction as a \
 short-term measure for enhanced security.\n\n\
 The next software release will provide more robust protection tools so that offers with this risk profile can be traded again.
```